http/server_test.ts

```diff
@@ -155,7 +155,7 @@ Deno.test(
 
     try {
       assertEquals(server.addrs.length, 3);
-      assertEquals(server.addrs[0].transport, "tcp");
+      assertEquals((server.addrs[0] as Deno.NetAddr).transport, "tcp");
```

Comment:
We can use non-null assertions here.

Reply:
For the other parameters is used the casting, so I have to modify them too? Just for code coherence